commons/src/main/scala/com/lynbrookrobotics/potassium/commons/drivetrain/UnicycleCoreTasks.scala

```diff
@@ -175,4 +178,53 @@ trait UnicycleCoreTasks {
       drive.resetToDefault()
     }
   }
+
+  class CorrectOffsetWithLatency(timestampedOffset: Signal[(Angle, Time)], tolerance: Angle)
+    (implicit drive: Drivetrain,
+      hardware: DrivetrainHardware,
+      props: Signal[DrivetrainProperties]) extends FiniteTask {
+
+    val positionSlide = hardware.turnPosition.toPeriodic.slidingWithTime(
+      20, hardware.turnPosition.get
+    ).toPollingSignal(Milliseconds(5))(drive.clock)
```

Comment:
If possible, don't use a polling signal and instead bring in values from this signal using `zip`. Polling signals really shouldn't be used unless you are producing sensor data, because they automatically assign a clock to the value and don't allow the component that's using the signal for control data to configure the update rate.

Also, polling signals can cause lots of leaks, because they never stop updating even if no one is using them.